ci-operator/step-registry/redhat-et/microshift/e2e/wait-for-cluster-up/redhat-et-microshift-e2e-wait-for-cluster-up-commands.sh

```diff
@@ -77,16 +77,10 @@ LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute scp \
   --zone "${GOOGLE_COMPUTE_ZONE}" \
   --recurse "${HOME}"/wait_for_node_ready.sh rhel8user@"${INSTANCE_PREFIX}":~/wait_for_node_ready.sh
 
-LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute scp \
-  --quiet \
-  --project "${GOOGLE_PROJECT_ID}" \
-  --zone "${GOOGLE_COMPUTE_ZONE}" \
-  --recurse "${HOME}"/wait_for_node_ready.sh rhel8user@"${INSTANCE_PREFIX}":~/wait_for_node_ready.sh
-
 LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
   --zone "${GOOGLE_COMPUTE_ZONE}" \
   rhel8user@"${INSTANCE_PREFIX}" \
-  --command 'sudo systemctl enable --now microshift.service'
+  --command 'sudo systemctl enable --now microshift.service && /home/rhel8user/wait_for_node_ready.sh'
```

Comment:
The server enablement is duped with:
https://github.com/openshift/release/blob/97cde1fd68bddb3480fef3fdcc116c865c242532/ci-operator/step-registry/redhat-et/microshift/e2e/pre-container-install/redhat-et-microshift-e2e-pre-container-install-commands.sh#L131-L134

Reply:
noted, we'll fix in follow-up, since we want to unblock the repo asap